src/test/java/roomescape/domain/ReservationPolicyTest.java

```diff
@@ -0,0 +1,82 @@
+package roomescape.domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import roomescape.TestFixture;
+import roomescape.exception.UnAvailableReservationException;
+
+class ReservationPolicyTest {
+
+    private static final String PAST_RESERVATION_MESSAGE = "지난 날짜와 시간에 대한 예약은 불가능합니다.";
+    private static final String TOO_SOON_RESERVATION_MESSAGE = "예약 시간까지 10분도 남지 않아 예약이 불가합니다.";
+    private static final String DUPLICATE_RESERVATION_MESSAGE = "테마에 대해 날짜와 시간이 중복된 예약이 존재합니다.";
+
+    @ParameterizedTest
+    @CsvSource({
+            "2025-04-23T12:00, 2025-04-23T11:59",
+            "2025-04-23T12:00, 2025-04-22T12:00"
+    })
+    void 지난_날짜_예약은_불가능하다(String fixedNowStr, String reservationDateTimeStr) {
```

Comment:
```suggestion
    @ParameterizedTest
    @CsvSource({
            "2025-04-23T12:00, 2025-04-23T11:59",
            "2025-04-23T12:00, 2025-04-22T12:00"
    })
    void 지난_날짜_예약은_불가능하다(LocalDateTime fixedNowStr, LocalDateTime reservationDateTimeStr) {
```

LocalDateTime 으로 받을 수 있을 것 같아요.

Reply:
그렇네요 훨씬 간결해질 것 같습니다.
반영했습니다!